MHVLib/Connection/MHVConnection.m

```diff
@@ -257,7 +257,21 @@ - (void)executeRestRequest:(MHVHttpServiceRequest *)request
             return;
         }
         
-        if (error)
+        if (response.hasError)
```

Comment:
if (error) is fine if there is a protocol error - timed out, network not available, etc. If there's a bad response - 400, 500, etc - then hasError is true and we need to look at more details. Two questions I have here - is it safe to put the whole responseAsString in the error? and maybe we should create something more meaningful than MHVUnknown error?